pkg/resource/v1/certconfig/create_test.go

```diff
@@ -75,7 +75,7 @@ func Test_ApplyCreateChange_Creates_createChange(t *testing.T) {
 }
 
 func Test_ApplyCreateChange_Does_Not_Make_API_Call_With_Empty_CreateChange(t *testing.T) {
-	logger, err := micrologger.New(micrologger.DefaultConfig())
+	logger, err := micrologger.New(micrologger.Config{})
```

Comment:
FYI @tuommaki we can usually use `microloggertest.New()` in most tests. I am not sure we need a real logger here.